app/controllers/web/console.php

```diff
@@ -5,10 +5,11 @@
 
 App::get('/console/*')
     ->alias('/')
+    ->alias('auth/*')
```

Comment:
Is this on purpose? Looking at code around, I would expect `/` at beginning

Reply:
I think it works both ways ( with and without the leading / ) I had tested it and it seemed to work